run_perf_tests.py

```diff
@@ -307,25 +301,56 @@ def analyze_data_files(dest_dir):
         return 1
 
 
-def get_testcase_name(dest_dir, full_filename):
-    dump_results = parse_dump_file(full_filename)
-    testcase_name = dump_results['Testname']
-    testcase_names.append(testcase_name)
-
-
-def xml(dest_dir, jank_perc, execution_time):
+def xml(dest_dir,test_data_dir):
     xml_file_dir = os.path.join(dest_dir, 'app/build/outputs/androidTest-results/connected')
```

Comment:
Do not use platform-specific directory separator. Always use `os.path.join` to construct pathnames.